src/main/java/com/agilethought/atsceapi/validator/Validator.java

```diff
@@ -0,0 +1,7 @@
+package com.agilethought.atsceapi.validator;
+
+import com.agilethought.atsceapi.model.User;
+
+public interface Validator<E> {
+	public void isValid(E object);
```

Comment:
I prefer method name as `validate(E object)`, because with isValid() we can infer this method could return a boolean response. in the other hand validate() sounds like a process.
Take this a suggestion.